pkg/util/normalizer/markdown.go

```diff
@@ -0,0 +1,145 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package normalizer
```

Comment:
Should we make kubectl use this code?
Can be done as a follow-up

Reply:
if this is not in contrast with the planned repository split, I guess we can create an issue for tracking this. But this is out of scope

Reply:
Yeah,  let's create an issue and follow up :+1:

Reply:
Can you please add a comment where this code is copied from and what it should be unified with so we don't lose track here?